views/default/manage_packages.pdt

```diff
@@ -277,7 +277,7 @@
         <div class="inner">
             <div class="button_row">
                 <?php
-                $this->Form->fieldSubmit('save', $this->_('Namesilo.manage.create_packages.create_packages_btn', true), ['class' => 'btn btn-primary pull-right']);
+                $this->Form->fieldSubmit('save', $this->_('Namesilo.manage.create_packages.save_packages_btn', true), ['class' => 'btn btn-primary pull-right']);
```

Comment:
If we're renaming, we should probably change this and language definitions like it to Namesilo.manage.manage_packages...